### PR TITLE
[risk=low] RW-1691 Added deploy scripts

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Added deploy scripts.

### CODE REVIEW

File: deploy/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      # Required: a GitHub release git tag, e.g. v1-0-rc1
+      - WORKBENCH_VERSION
+    volumes:
+      # Cache the codebase and gradle for deployment only.
+      - gradle-cache:/.gradle
+      - workbench:/home/circleci/workbench
+      - ./bootstrap-docker.sh:/bootstrap-docker.sh
+
+volumes:
+  workbench:
```
Comment (calbach) [inline deploy/docker-compose.yaml]: nit: workbench -> data-browser

File: deploy/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  deploy:
+    image: allofustest/workbench:buildimage-0.0.13
+    entrypoint: /bootstrap-docker.sh
+    user: circleci
+    environment:
+      # Required: a GitHub release git tag, e.g. v1-0-rc1
+      - WORKBENCH_VERSION
```
Comment (calbach) [inline deploy/docker-compose.yaml]: nit: DATA_BROWSER_VERSION here and above

File: deploy/libproject/jirarelease.rb [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    jira_creds = Common.new.capture_stdout(['gsutil', 'cat', gcs_uri])
+    unless jira_creds
+      raise RuntimeError.new "failed to read JIRA login from '#{gcs_uri}'"
+    end
+    jira_json = JSON.parse(jira_creds)
+    return JiraReleaseClient.new(jira_json['username'], jira_json['password'])
+  end
+
+  def ticket_summary(tag)
+    return "Release tracker: Workbench #{tag}"
```
Comment (calbach) [inline deploy/libproject/jirarelease.rb]: Data Browser

**Review by calbach — COMMENTED** [review]
Nice - thanks. Were you able to test this successfully using the --dry-run flag?

File: deploy/docker-compose.yaml (hunk 2 above)
Comment (SrushtiGangireddy) [reply deploy/docker-compose.yaml]: Done.

File: deploy/libproject/jirarelease.rb (hunk 3 above)
Comment (SrushtiGangireddy) [reply deploy/libproject/jirarelease.rb]: Done.

File: deploy/docker-compose.yaml (hunk 1 above)
Comment (SrushtiGangireddy) [reply deploy/docker-compose.yaml]: Done.

**Review by SrushtiGangireddy — COMMENTED** [review]
I tested the script with --dry-run and it ran successfully.

File: deploy/docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    image: allofustest/workbench:buildimage-0.0.13
+    entrypoint: /bootstrap-docker.sh
+    user: circleci
+    environment:
+      # Required: a GitHub release git tag, e.g. v1-0-rc1
+      - DATA_BROWSER_VERSION
+    volumes:
+      # Cache the codebase and gradle for deployment only.
+      - gradle-cache:/.gradle
+      - data-browser:/home/circleci/workbench
```
Comment (SrushtiGangireddy) [inline deploy/docker-compose.yaml]: Not sure if /home/circle/data-browser is present. 
Do i need to change this ?

Comment (calbach) [reply deploy/docker-compose.yaml]: Yes - 'data-browser' will be there, this is the name of the cloned github directory.

**Review by SrushtiGangireddy — COMMENTED** [review]
Done. pushed in change. thanks.

File: deploy/bootstrap-docker.sh [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+sudo chmod g+r /creds/sa-key.json
+sudo chown -R circleci /.gradle
+
+if [[ ! -d ~/data-browser/.git ]]; then
+  sudo git clone https://github.com/all-of-us/data-browser ~/data-browser
+  sudo chown -R circleci ~/data-browser
+fi
+cd ~/data-browser
+git fetch
+git checkout "${WORKBENCH_VERSION}"
```
Comment (calbach) [inline deploy/bootstrap-docker.sh]: DATA_BROWSER_VERSION

**Review by calbach — COMMENTED** [review]
One last issue, then it looks good.

File: deploy/bootstrap-docker.sh (hunk 5 above)
Comment (SrushtiGangireddy) [reply deploy/bootstrap-docker.sh]: Thought i changed it already. Sorry missed it somehow. Fixed now.

**Comment by SrushtiGangireddy** [comment]
I do not know what's up with the indentation in readme. The mark down looks good to me.

**Review by calbach — APPROVED** [review]
I've disabled that - workbench doesn't have that markdown linter in place. I also disabled the other Ruby complexity lint, which we disabled in Workbench as well. Codacy should be green on a rerun (I clicked reanalyse to verify)

**Comment by SrushtiGangireddy** [comment]
Cool thanks, merging in.

**Pull request merged**